openmdao/components/tests/test_submodel_comp.py

```diff
@@ -452,15 +454,83 @@ def test_submodel_comp(self):
 
         par = model.add_subsystem('par', om.ParallelGroup())
 
-        par.add_subsystem('subprob1', om.SubmodelComp(problem=om.Problem(model=FanOut()),
-                                                      inputs=['p.x'], outputs=['comp2.y', 'comp3.y']))
-        par.add_subsystem('subprob2', om.SubmodelComp(problem=om.Problem(model=FanIn()),
-                                                      inputs=['p1.x1', 'p2.x2'], outputs=['comp3.y']))
+        par.add_subsystem('subprob1', om.SubmodelComp(problem=om.Problem(model=FanOutGrouped()),
+                                                      inputs=['iv.x'], outputs=['c2.y', 'c3.y'],
+                                                      do_coloring=True))
+        par.add_subsystem('subprob2', om.SubmodelComp(problem=om.Problem(model=FanInGrouped()),
+                                                      inputs=['*'], outputs=['c3.y'],
+                                                      do_coloring=True))
 
         p.setup(force_alloc_complex=True)
         p.run_model()
         assert_check_partials(p.check_partials(method='cs', out_stream=None))
 
+    # @unittest.skip("Unskip this after vars under ParallelGroups work with SubmodelComp")
```

Comment:
Should this be commented?

Reply:
I just removed it.  At one point when I was trying to get distrib vars to work, both parallel group vars and distrib vars were broken, but when I backed out all of those changes, the parallel vars worked, so right now the only thing that doesn't work is the distrib vars.  Actually Ken found one more issue (no support for unit conversions between inner and outer model), but I'll fix that in a follow-on PR.